### ISSUE TITLE
[Bug]: Javascript error and then Pluto app just spins

### ISSUE THREAD

**Issue by stefanjwojcik** [JuliaPluto/PlutoDesktop#64]
### Contact Details

swojcik@nationalgridrenewables.com

### Process where you encountered the bug

Something else

### Level of issue

Does not cause crash, but is a major issue [severe:s2]

### What happened?

I was using Pluto desktop happily for a number of months, but at some point when loading I began getting a Javascript error, after which the app would just give the loading screen continuously and never successfully open. I can't even open Pluto to troubleshoot, so unfortunately I can't give much detail beyond the error and my system details. I tried uninstalling and re-installing the app, but that did not resolve the issue. 

I'm using Pluto Desktop release: [v0.0.2](https://github.com/JuliaPluto/PlutoDesktop/tree/v0.0.2). 

OS  - Windows 11 Pro
Processor - 12th Gen Intel(R) Core(TM) i7-12800HX   2.00 GHz
Installed RAM - 64.0 GB (63.7 GB usable)

Error: 
![pluto_error](https://user-images.githubusercontent.com/12849203/230604443-eeb1ef0d-c1e3-483a-bf03-0bc38ec99a6e.png)

### Version

0.0.2

### Relevant log output from C:\Users\{USER_NAME}\AppData\Roaming\pluto\logs\main.log

```shell
Stacktrace:
2023-04-07 06:32:59.094 error julia > 
 [1] 
2023-04-07 06:32:59.232 error julia > require(
2023-04-07 06:32:59.282 error julia > into::
2023-04-07 06:32:59.318 error julia > Module, mod::
2023-04-07 06:32:59.322 error julia > Symbol)
   @ 
2023-04-07 06:32:59.377 error julia > Base 
2023-04-07 06:32:59.388 error julia > .\
2023-04-07 06:32:59.427 error julia > loading.jl:967
2023-04-07 06:32:59.430 error julia > 
in expression starting at C:\Users\swojcik\.julia\config\startup.jl:14

2023-04-07 06:32:59.455 info julia > child process exited with code 1
2023-04-07 06:46:16.870 verbose julia > Killing Pluto process.
2023-04-07 06:47:00.578 verbose general > ---------- NEW SESSION ----------
2023-04-07 06:47:00.604 verbose general > Application Version: 0.0.2
2023-04-07 06:47:00.609 verbose general > CONFIG STORE: {
  __internal__: { migrations: { version: '0.0.2' } },
  'IMPORTANT-NOTE': 'This file is used for internal configuration. Please refrain from editing or deleting this file.',
  'JULIA-PATH': 'C:\\Program Files\\Pluto\\resources\\assets\\julia-1.7.3\\bin\\julia.exe',
  'PLUTO-PRECOMPILED': 'C:\\Program Files\\Pluto\\resources\\assets\\pluto-sysimage.so'
}
2023-04-07 06:47:00.622 verbose general > USER STORE: {}
2023-04-07 06:47:00.731 info general > Arguments received: { _: [], '$0': 'Pluto.exe' }
2023-04-07 06:47:01.078 info general > Julia found at: C:\Program Files\Pluto\resources\assets\julia-1.7.3\bin\julia.exe
2023-04-07 06:47:01.106 info general > LAUNCHING
 project: C:\Users\swojcik\AppData\Roaming\pluto\project\ 
notebook: undefined
2023-04-07 06:47:01.122 verbose general > Executing C:\Program Files\Pluto\resources\assets\julia-1.7.3\bin\julia.exe with options --project=C:\Users\swojcik\AppData\Roaming\pluto\project\ --sysimage=C:\Program Files\Pluto\resources\assets\pluto-sysimage.so,C:\Program Files\Pluto\resources\assets\pluto_no_update.jl
2023-04-07 06:47:01.613 error julia > ERROR: 
2023-04-07 06:47:01.671 error julia > LoadError: 
2023-04-07 06:47:02.094 error julia > ArgumentError: 
2023-04-07 06:47:02.097 error julia > Package Revise not found in current path:
- Run `import Pkg; Pkg.add("Revise")` to install the Revise package.

Stacktrace:
 [1] 
2023-04-07 06:47:02.237 error julia > require(
2023-04-07 06:47:02.284 error julia > into::
2023-04-07 06:47:02.319 error julia > Module, 
2023-04-07 06:47:02.322 error julia > mod::Symbol)
   @ 
2023-04-07 06:47:02.371 error julia > Base .\
2023-04-07 06:47:02.416 error julia > loading.jl:967
2023-04-07 06:47:02.419 error julia > 
in expression starting at C:\Users\swojcik\.julia\config\startup.jl:14

2023-04-07 06:47:02.444 info julia > child process exited with code 1
2023-04-07 06:52:05.168 verbose julia > Killing Pluto process.
```

### Relevant output from C:\Users\{USER_NAME}\AppData\Roaming\pluto\config.json

```json
{
	"__internal__": {
		"migrations": {
			"version": "0.0.2"
		}
	},
	"IMPORTANT-NOTE": "This file is used for internal configuration. Please refrain from editing or deleting this file.",
	"JULIA-PATH": "C:\\Program Files\\Pluto\\resources\\assets\\julia-1.7.3\\bin\\julia.exe",
	"PLUTO-PRECOMPILED": "C:\\Program Files\\Pluto\\resources\\assets\\pluto-sysimage.so"
}
```

### Relevant output from C:\Users\{USER_NAME}\AppData\Roaming\pluto\userConfig.json

```json
{}
```

**Comment by fonsp** [comment]
Did you recently [add Revise to your `~/.julia/config/startup.jl` file](https://timholy.github.io/Revise.jl/stable/config/#Using-Revise-by-default-1)?

**Comment by stefanjwojcik** [comment]
Hmm, it's possible I added it after installing the Pluto Desktop App. Is it best practice to avoid this? Happy to close if this is expected in that case.

**Comment by fonsp** [comment]
Okay! It's not best practice to avoid this, don't worry!

This should be fixed by #57, since the separate DEPOT won't have the user's default startup.jl. (That PR has been mergen but not released yet)
